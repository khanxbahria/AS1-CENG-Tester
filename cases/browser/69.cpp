#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("l", "ppp","uuu");
Tab B = Tab("ff", "t","n");
Tab C = Tab("aaa", "qq","qq");
Tab D = Tab("kk", "r","x");
Tab E = Tab("ttt", "uu","qqq");
Tab F = Tab("vvv", "z","x");
Tab G = Tab("www", "ii","xx");
Tab H = Tab("r", "m","v");
Tab I = Tab("fff", "vvv","r");
Tab J = Tab("gg", "q","e");
Tab K = Tab("w", "nn","ooo");
Tab L = Tab("ooo", "ss","q");
Tab M = Tab("ll", "jjj","rr");
Tab N = Tab("aa", "m","q");
Tab O = Tab("ttt", "zzz","www");
Tab P = Tab("oo", "h","cc");
Tab Q = Tab("mm", "eee","f");
Tab R = Tab("zz", "tt","bbb");
Tab S = Tab("bbb", "ttt","bbb");
Tab T = Tab("vv", "s","lll");
Tab U = Tab("l", "cc","z");
Tab V = Tab("sss", "w","x");
Tab W = Tab("a", "a","ppp");
Tab X = Tab("l", "nn","ss");
Tab Y = Tab("ii", "ll","ppp");
Tab Z = Tab("rrr", "zz","fff");
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(A);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.newTab(B);
a.newTab(C);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(0);
b.newTab(D);
c.newTab(E);
d.newTab(F);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.newTab(G);
d.changeActiveTabTo(4);
a.newTab(H);
b.changeActiveTabTo(0);
c.newTab(I);
d.newTab(J);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(K);
d.newTab(L);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(0);
b.newTab(M);
c.moveActiveTabTo(0);
d.newTab(N);
a.newTab(O);
b.newTab(P);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.newTab(Q);
b.newTab(R);
c.newTab(S);
d.changeActiveTabTo(1);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.newTab(T);
d.moveActiveTabTo(1);
a.newTab(U);
b.changeActiveTabTo(5);
c.newTab(V);
d.newTab(W);
a.newTab(X);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(Y);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.newTab(Z);
d.newTab(Z);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
a.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
c.moveActiveTabTo(3);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
a.changeActiveTabTo(5);
d.moveActiveTabTo(5);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(5);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
c.moveActiveTabTo(0);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->print();
brows->switchToWindow(7);
brows->newWindow();
brows->switchToWindow(5);
brows->mergeAllWindows();
brows->getWindow(8);
brows->switchToWindow(5);
brows->newWindow();
brows->getWindow(0);
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->closeWindow();
brows->print();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(3);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->switchToWindow(6);
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->getWindow(0);
brows->switchToWindow(1);
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->print();
brows->print();
brows->getWindow(0);
brows->closeEmptyWindows();
brows->getWindow(1);
brows->getWindow(3);
brows->closeWindow();
brows->getWindow(8);
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(2);
brows->newWindow();
brows->closeWindow();
brows->closeWindow();
brows->getWindow(3);
brows->newWindow();
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(1);
brows->print();
brows->closeEmptyWindows();
brows->print();
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->switchToWindow(6);
brows->newWindow();
brows->print();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(6);
brows->getWindow(9);
brows->closeWindow();
brows->print();
brows->closeEmptyWindows();
brows->switchToWindow(8);
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->newWindow();
brows->getWindow(8);
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(4);
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(6);
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(0);
brows->newWindow();
brows->getWindow(8);
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(9);
brows->getWindow(9);
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(2);
delete brows;
}
	return 0;
}