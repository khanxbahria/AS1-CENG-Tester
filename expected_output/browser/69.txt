The active tab is tab 32








m
r
v

ll
ii
ppp

zzz
ttt
www

eee
mm
f

cc
l
z

nn
l
ss





ppp
l
uuu

qq
aaa
qq





r
kk
x

h
oo
cc

tt
zz
bbb

jjj
ll
rr

nn
w
ooo

uu
ttt
qqq





zz
rrr
fff

ii
www
xx

ttt
bbb
bbb

s
vv
lll

w
sss
x

vvv
fff
r

a
a
ppp

t
ff
n





z
vvv
x

q
gg
e

zz
rrr
fff

ss
ooo
q

m
aa
q

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 33












m
r
v

ll
ii
ppp

zzz
ttt
www

eee
mm
f

cc
l
z

nn
l
ss





ppp
l
uuu

qq
aaa
qq





r
kk
x

h
oo
cc

tt
zz
bbb

jjj
ll
rr

nn
w
ooo

uu
ttt
qqq





zz
rrr
fff

ii
www
xx

ttt
bbb
bbb

s
vv
lll

w
sss
x

vvv
fff
r

a
a
ppp

t
ff
n





z
vvv
x

q
gg
e

zz
rrr
fff

ss
ooo
q

m
aa
q

The active tab is tab 0




The active tab is tab 33












m
r
v

ll
ii
ppp

zzz
ttt
www

eee
mm
f

cc
l
z

nn
l
ss





ppp
l
uuu

qq
aaa
qq





r
kk
x

h
oo
cc

tt
zz
bbb

jjj
ll
rr

nn
w
ooo

uu
ttt
qqq





zz
rrr
fff

ii
www
xx

ttt
bbb
bbb

s
vv
lll

w
sss
x

vvv
fff
r

a
a
ppp

t
ff
n





z
vvv
x

q
gg
e

zz
rrr
fff

ss
ooo
q

m
aa
q

The active tab is tab 33












m
r
v

ll
ii
ppp

zzz
ttt
www

eee
mm
f

cc
l
z

nn
l
ss





ppp
l
uuu

qq
aaa
qq





r
kk
x

h
oo
cc

tt
zz
bbb

jjj
ll
rr

nn
w
ooo

uu
ttt
qqq





zz
rrr
fff

ii
www
xx

ttt
bbb
bbb

s
vv
lll

w
sss
x

vvv
fff
r

a
a
ppp

t
ff
n





z
vvv
x

q
gg
e

zz
rrr
fff

ss
ooo
q

m
aa
q

The active tab is tab 33












m
r
v

ll
ii
ppp

zzz
ttt
www

eee
mm
f

cc
l
z

nn
l
ss





ppp
l
uuu

qq
aaa
qq





r
kk
x

h
oo
cc

tt
zz
bbb

jjj
ll
rr

nn
w
ooo

uu
ttt
qqq





zz
rrr
fff

ii
www
xx

ttt
bbb
bbb

s
vv
lll

w
sss
x

vvv
fff
r

a
a
ppp

t
ff
n





z
vvv
x

q
gg
e

zz
rrr
fff

ss
ooo
q

m
aa
q

The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
The browser is empty
The browser is empty
The active tab is tab 0




